index.html

```diff
@@ -2776,8 +2776,9 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="directory">
 			<rdef>directory</rdef>
 			<div class="role-description">
-				<p>A list of references to members of a group, such as a static table of contents.</p>
-				<p>Authors SHOULD use this <a>role</a> for a static table of contents, whether linked or unlinked. This includes tables of contents built with lists, including nested lists. Dynamic tables of contents, however, might use a <rref>tree</rref> role instead.</p>
+				<p>[Deprecated in ARIA 1.2] A list of references to members of a group, such as a static table of contents.</p>
+				<p class="note">As exposed by accessibility APIs the <code>directory</code> <a>role</a> provides no benefits to users over using the <code>list</code> <a>role</a>. Authors are advised to treat <code>directory</code> as deprecated and to use <code>list</code>, or a host language's equivalent semantics instead.</p>
```

Comment:
Fix missing comma and clarify wording. Recommend:

```suggestion
				<p class="note">As exposed by accessibility APIs, the <code>directory</code> <a>role</a> is essentially equivalent to the <code>list</code> <a>role</a>. So, using <code>directory</code> does not provide any additional benefits to assistive technology users. Authors are advised to treat <code>directory</code> as deprecated and to use <code>list</code>, or a host language's equivalent semantics instead.</p>
```